File: OCH_Tao/ABC/B/287.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [N, M] = input.shift().split(" ").map(x => parseInt(x, 10));
const S = input.splice(0, N);
const T = new Set(input);
let cnt = 0;
for (const i of S) {
  if (T.has(i.slice(3))) {
    cnt++;
  }
}
console.log(cnt);

File: OCH_Tao/ABC/B/287.py
N,M = map(int,input().split())
S = []
for i in range(N):
  S.append(input())
T = set()
for i in range(M):
  T.add(input())
cnt = 0
for i in S:
  if i[3:] in T:
    cnt+=1
print(cnt)